JS-Algorithms/challenges.js

```diff
@@ -39,9 +49,23 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 ***** */
 
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
+const MODIFIED_COUNTRY_NAMES = new Array();
 
 const circularArray = (index) => {
   // YOUR CODE HERE...
+  if(index >= 5) index = index % 5;
```

Comment:
The purpose of this operation is to obtain an index of the array if the index given is greater than or equal to 5 and exceeds the array. For example if the index were 7, the module would be (7 % 5 = 2) which would be equivalent to the index 2 in our current array.

Reply:
It would be better to either throw an error or return null instead. The problem is that the user is adding an index greater or equal to `5` and we are taking it as if it was a `2` instead which wouldn't make sense from a user perspective.